ROADMAP.md

```diff
@@ -0,0 +1,33 @@
+# YDB Embedded UI Roadmap
+## Legend
+We use the following symbols as abbreviations:
+
+1. ㉕ - feature appeared in the Roadmap for 2025;
+1. ㉔ - feature appeared in the Roadmap for 2024;
+1. ✅ - feature has been released;
+1. 🚧 - feature is partially available and is under development;
```

Comment:
Does this mean "in progress"? Or it mean feature exists, but works unstable or can be even turned in by feature-flag? Because currently you marked with this some of the points, that currently aren't added into UI - Autocompletion, Charts, Hot keys. Probably they should be 🔥, and the only point with 🚧 is Capacity diagnostics